WD: Mutual Aid for Writers proj

rosewms opened the pull request:
Atlantic 2024 Writing Day project, Mutual Aid for Documentarians/Tech Writers Job Hunters.

Steps before publishing:

- [x]  Staff review of project to confirm Writing Day suitability
- [x]  WD project lead/organizer review to ensure project is described properly
- [x]  Final staff review before publishing

<!-- readthedocs-preview writethedocs-www start -->
----
📚 Documentation preview 📚: https://writethedocs-www--2202.org.readthedocs.build/

<!-- readthedocs-preview writethedocs-www end -->

rosewms committed:
WD: Mutual Aid for Writers proj

rainbowkenz reviewed (changes requested):
“Woodbridge” is missing the final “e”. :) Otherwise, looks good.

plaindocs commented on docs/conf/atlantic/2024/writing-day.rst:
```diff
@@ -127,3 +127,52 @@ Project list
+Mutual Aid for Tech Writer/Documentarian Job Hunters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Join project organizer, Kenzie Woodbridg (they/them), during the second Writing Day session.
```
```suggestion
Join project organizer, Kenzie Woodbridge (they/them), during the second Writing Day session.
```

plaindocs committed:
Update docs/conf/atlantic/2024/writing-day.rst

rosewms committed:
Merge branch 'main' into wd-mutual-aid-proj

rosewms committed:
Removed additional line breaks

Kenzie's project is ready to publish, it will be the first project out. Edited to remove additional lines.
